site2/docs/concepts-messaging.md

```diff
@@ -20,11 +20,12 @@ Value / data payload | The data carried by the message. All Pulsar messages cont
 Key | Messages are optionally tagged with keys, which is useful for things like [topic compaction](concepts-topic-compaction.md).
 Properties | An optional key/value map of user-defined properties.
 Producer name | The name of the producer who produces the message. If you do not specify a producer name, the default name is used. 
+Topic name | The name of the topic that the message is published to.
```

Comment:
```suggestion
Topic name | The name of the topic to which the message is published.
```

Reply:
@Anonymitaet Is it a restrictive clause and only "that" can be used?

Reply:
Here is "use prepositions with relative pronouns" rather than "restrictive clause (that) vs non-restrictive clause (which) - American English"

### Use prepositions with relative pronouns
Prepositions (In, On, At, To) are used with relative pronouns(Which) to indicate a relationship between two clauses when referring to a particular subject. 
eg. 
The town to which we were driving was 50 ks away.
This is the card to which I was referring.

### that VS which (restrictive clause - American English)
eg. 
Set the `service_url` to the value which is obtained through Step 1. ❌
Set the `service_url` to the value that is obtained through Step 1. ✅

"that" is used with a restrictive clause. "which" is used with a non-restrictive clause. In these cases, the information is essential to the meaning (a restrictive clause), so we should use "that".

Reply:
Thanks for the clarification and I see your point. I will add your explanations to the style guide. 
But I wonder both are correct in this case (will turn to our editors for a double-check).

Reply:
OK, thanks.